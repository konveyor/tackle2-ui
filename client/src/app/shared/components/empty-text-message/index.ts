export { EmptyTextMessage } from "./empty-text-message";
