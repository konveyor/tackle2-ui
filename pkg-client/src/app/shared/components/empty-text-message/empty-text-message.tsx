import React from "react";
import { Text } from "@patternfly/react-core";

export interface EmptyTextMessageProps {
  message: string;
}

export const EmptyTextMessage: React.FC<EmptyTextMessageProps> = ({
  message,
}) => {
  return (
    <Text className="pf-u-color-200 pf-u-font-weight-light">
      <i>{message}</i>
    </Text>
  );
};
