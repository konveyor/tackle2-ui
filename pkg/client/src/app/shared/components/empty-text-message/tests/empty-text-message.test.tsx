import React from "react";
import { shallow } from "enzyme";
import { EmptyTextMessage } from "../empty-text-message";

describe("EmptyTextMessage", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<EmptyTextMessage message="my custom message" />);
    expect(wrapper).toMatchSnapshot();
  });
});
