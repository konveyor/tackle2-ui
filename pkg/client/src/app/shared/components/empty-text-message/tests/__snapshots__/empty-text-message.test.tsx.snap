// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyTextMessage Renders without crashing 1`] = `
<Text
  className="pf-u-color-200 pf-u-font-weight-light"
>
  <i>
    my custom message
  </i>
</Text>
`;
